test/posts.test.js

```diff
@@ -0,0 +1,52 @@
+const request = require('supertest');
+const app = require('../src/backend/web/app');
+const { addPost } = require('../src/backend/utils/storage');
+
+describe('test /posts endpoint', async () => {
+  const defaultItems = 30;
+  const requestedItems = 50;
+  const maxItems = 100;
+
+  const posts = [...Array(150).keys()].map(guid => {
```

Comment:
This is really smart.  Later on we should extract this out into a testing lib we can use in other tests.